### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by chrisdswift** [ElectroTechnique/electrotechnique.github.io#10] [Software Issues]
I propose a section for users to add requests for future features.  My first suggestion is:

To add a mode whereby the system can be booted into USB mass storage mode so that patch files on the SD card can be maintained from a connected host computer in a similar way to the PreenFM2 synth [https://ixox.fr/preenfm2/](url)

**Comment by Hububalli** [comment]
Yeah these are fantastic ideas, both the topic and the suggestion. 

I added a flap in the case to access the card but it is crude. Would be much nicer to be able to access via USB.

**Comment by Electrotechnique** [comment]
Yes, that would be a great way to access the patches. However, currently I don't think that the USB client port on the Teensy 4 will allow itself to appear as mass storage (_MTP disk(experimental)_ in USB Type?) and MIDI/digital audio/serial. It could possibly be put into a mass storage mode during startup and then restarted back to normal. It requires some investigation.

**Comment by RyzArgos** [comment]
How about adding a 1/8" patch bay for semi-modular capability?
